docs/messaging/gleanplumb.mdx

```diff
@@ -735,20 +735,26 @@ so boolean attributes should be simulated with strings or integers.
 }>
 <TabItem value="fenix">
 
-Attribute | Type | Description
-----------|------|------------
-`is_default_browser_string`                 | string   | Should change to boolean on fixing [`==`][bool-equ] and [`!`][bool-not] |
-`date_string`                               | string   | In YYYY-MM-DD format                                                    |
-`number_of_app_launches`                    | int      | Indicates how many times the app has been launched.                     |
+Attribute | Type | Description | Versions |
+----------|------|-------------|----------|
+`is_default_browser`                        | boolean  | JEXL.rs does not implement [boolean negation `!`][bool-not]             |  |
+`date_string`                               | string   | In YYYY-MM-DD format                                                    |  |
+`number_of_app_launches`                    | int      | Indicates how many times the app has been launched.                     |  |
+`adjust_campaign`                           | string?  | The campaign id parameter as derived by Adjust                          | v111 |
+`adjust_network`                            | string?  | The network parameter as derived by Adjust                              | v111 |
+`adjust_ad_group`                           | string?  | The Ad Group parameter as derived by Adjust                             | v111 |
+`adjust_creative`                           | string?  | The Creative parameter as derived by Adjust                             | v111 |
+`are_notifications_enabled`                 | boolean  | JEXL.rs does not implement [boolean negation `!`][bool-not]             | v111 |
 
 It is possible this table is out of date. The definitive source of truth for this in [the code][fenix-attributes-eng] itself.
 
 </TabItem>
 <TabItem value="fxios">
 
-Attribute | Type | Description
-----------|------|------------
-`date_string`                        | string   | In YYYY-MM-DD format |
+Attribute | Type | Description | Versions |
+----------|------|-------------|----------|
+`date_string`                               | string   | In YYYY-MM-DD format                                                    |
+`is_default_browser`                        | boolean  | JEXL.rs does not implement [boolean negation `!`][bool-not]             |
```

Comment:
I'd recommend swapping the order of these two to match the order on the Fenix tab (or vice versa)